docs/en/operations/utils/clickhouse-benchmark.md

```diff
@@ -25,17 +25,17 @@ Then pass this file to a standard input of `clickhouse-benchmark`.
 clickhouse-benchmark [keys] < queries_file
 ```
 
-## Keys
+## Keys {#clickhouse-benchmark-keys}
 
-- `-c  N`, `--concurrency=N` — Number of queries that `clickhouse-benchmark` sends simultaneously. Default value: 1.
+- `-c N`, `--concurrency=N` — Number of queries that `clickhouse-benchmark` sends simultaneously. Default value: 1.
 - `-d N`, `--delay=N` — Interval in seconds between intermediate reports (set 0 to disable reports). Default value: 1.
-- `-h WORD`, `--host=WORD` — Server host. Default value: `localhost`.
-- `-p N`, `--port=N` — Server port. Default value: 9000.
+- `-h WORD`, `--host=WORD` — Server host. Default value: `localhost`. For the [comparison mode](#clickhouse-benchmark-comparison-mode) you can use multiple `-h` keys.
```

Comment:
it's not clear how multiple `-h` and multiple `-p` are matched together. by index? cartesian product is built?

Reply:
Made clarification in the comparison mode description.